src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+
 public class Duke {
+    private static final String divider = "_______________________________\n";
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+
+    private static void addTask(String taskName){
+        Task currentTask = new Task(taskName);
+        tasks.add(currentTask);
+        System.out.println(divider + "added " + currentTask.description + "\n" + divider);
+    }
+
+    private static void listTask(){
+        System.out.println(divider);
+        System.out.println("Here are the tasks in your list:");
+        for(int i=0; i<tasks.size(); ++i){
+            System.out.println((i+1) + "." +
+                    tasks.get(i).toString()+ "\n");
+        }
+        System.out.println(divider);
+    }
+
+    private static void markTaskDone(int index){
+        Task currentTask = tasks.get(index-1);
+        currentTask.markAsDone();
+        System.out.println(divider + "Nice! I've marked this task as done:"+
+                "\n"+ currentTask.toString() +
+                "\n" + divider);
+    }
+
+    public static void addToDo(String input){
+        String taskName = input.substring(5);
```

Comment:
5 seems like a magical number here. Would it be better if you change it to get the index of the start of the task name?